python_ta/__init__.py

```diff
@@ -296,7 +299,7 @@ def reset_linter(
         (
             "pyta-template-file",
             {
-                "default": "template.html.jinja",
+                "default": os.path.join(TEMPLATES_DIR, "template.html.jinja"),
```

Comment:
It's not very common to have default values that are dynamically-set (in this case, based on the installation location for PythonTA). A more conventional approach is to use a default value like `''`, and then in the code which uses the value the dynamic `TEMPLATES_DIR` location is computed and the default template is used.